sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,26 @@
+function fetchAsyncSignals(mode) {
+        switch (mode) {
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab but found 8 spaces.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)